Library/Homebrew/cmd/uses.rb

```diff
@@ -64,10 +68,7 @@ def run
           opoo e
           used_formulae_missing = true
           # If the formula doesn't exist: fake the needed formula object name.
-          # This is a legacy use of OpenStruct that should be refactored.
-          # rubocop:disable Style/OpenStructUse
-          args.named.map { |name| OpenStruct.new name:, full_name: name }
-          # rubocop:enable Style/OpenStructUse
+          args.named.map { |name| UnavailableFormula.new name:, full_name: name }
```

Comment:
The signature of the `intersection_of_dependents` method below is wrong.

It should take `used_formulae: T::Array[T.any(Formula, UnavailableFormula)]`.

Reply:
Nice catch!